go/fn/runnerProcessor.go

```diff
@@ -30,45 +31,88 @@ type runnerProcessor struct {
 	fnRunner Runner
 }
 
+// EmptyFunctionConfig is a workaround solution to handle the case where kpt passes in a functionConfig placeholder
+// (Configmap with empty `data`) if user does not provide the actual FunctionConfig. Ideally, kpt should pass in an empty
+// FunctionConfig object.
+func EmptyFunctionConfig(o *KubeObject) bool {
+	data, found, err :=  o.NestedStringMap("data")
```

Comment:
Nit: I think there's an extra space here ... I'm guessing we don't have a `gofmt` test :-) 

I suggest something like this: https://github.com/justinsb/kweb/pull/10

Reply:
good catch! 
This repo has [CI that runs the `gofmt`](https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/.github/workflows/ci.yaml#L80) ([CI logs](https://github.com/GoogleContainerTools/kpt-functions-sdk/actions/runs/3244452086/jobs/5320646783#step:4:44)), but it needs some fix (i.e. after running the gofmt, CI should check the unstaged files  that runs the `gofmt` ).
Yeah, I like the  generate.sh script. I'm going to borrow that in this repo :D thank you!